packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -392,7 +404,14 @@ export class GcdsSelect {
             onFocus={() => this.gcdsFocus.emit()}
             onInput={e => this.handleInput(e, this.gcdsInput)}
             onChange={e => this.handleInput(e, this.gcdsChange)}
-            aria-invalid={hasError ? 'true' : 'false'}
+            aria-invalid={
+              inheritedAttributes['aria-invalid'] === 'true'
+                ? inheritedAttributes['aria-invalid']
+                : errorMessage
+                  ? 'true'
+                  : 'false'
+            }
+            part="input"
```

Comment:
Could we name this part `select` instead of `input` since it's not referring to an actual input?